wolftpm/tpm2_linux.h

```diff
@@ -0,0 +1,38 @@
+/* tpm2_linux.h
```

Comment:
Please add `tpm2_linux.h` to `wolftpm/include.am`.

Reply:
Thanks, it is added.